django/contrib/auth/models.py

```diff
@@ -159,11 +159,12 @@ def create_superuser(self, username, email, password, **extra_fields):
 
 
 # A few helper functions for common logic between User and AnonymousUser.
-def _user_get_all_permissions(user, obj):
+def _user_get_permissions(user, obj, from_name):
```

Comment:
I think I might prefer `scope` (or similar) to `from_name`. (As ever with naming...) 

Maybe make it kwarg-only so we'd call it like this: 

```
_user_get_permissions(self, obj, scope='user')
```

Otherwise, _"What's the string for?"_ — but this function is only used in this files, so maybe it's clear enough.

Reply:
I tried to keep close to the interface of `ModelBackend._gert_permissions()`. If this is changed here I think it should also be changed there. What do you think?

Reply:
Yep. That swings it for me. Leave it as is. Good one. Thanks.